### DISCUSSION TITLE
Version 0.22 is released

### DISCUSSION THREAD

**Discussion by mlopatkin** [mlopatkin/andlogview#356] [Announcements]
### New features
* Compatibility with Android 14 (#327)
* No restart required to change the ADB configuration (#197, #326)
* Support for the "long" format (#169)
* More reliable parsing of the multiline log messages (stacktraces) (#169)
* Improved extraction of process information for dumpstates (#336, #277, #88, #119, #85)

### Minor improvements and bugfixes
* "Disconnected" status now properly shown in the status bar (#330)
* "Delete" key now deletes bookmarks properly (#149)
* Proper file path copied when using status bar menu (#350)
* Fixed crash because of the missing preference directory on the very first run (#317)

## Contributors
I'd like to thank @nevack for their contributions.

**Full Changelog**: https://github.com/mlopatkin/andlogview/compare/0.21.1...0.22

SHA256 checksum: `92122a14f08e375e90c1dcde8c4149797e00918d3c36ded70904ddcf6bbe554c`

<hr /><em>This discussion was created from the release <a href='https://github.com/mlopatkin/andlogview/releases/tag/0.22'>0.22</a>.</em>

**Comment by maneesh0501** [comment]
Not working in macOS
Kindly provide complete procedure
